### PR TITLE
Feature: Add Kapacitor dependency

### PR DESCRIPTION
**Pull request by alpiquero**
* Add Kapacitor Helm chart 1.4.0 as dependency
* Add **kre_kapacitor.kap** file to `chronograf-config` configmap
* Update `values.yml`
* Update `scripts/krectl/cmd_deploy.sh`

### CODE REVIEW

File: helm/kre/templates/engine/chronograf/config.yml [hunk 1]
```diff
@@ -12,10 +12,21 @@ data:
...
+  kre_kapacitor.kap: |-
+    {
+      "id": "10001",
+      "srcID": "10000",
+      "name": "Kapacitor",
+      "url": "http://{{ include "kre-kapacitor.fullname" . }}:9092",
+      "active": true,
+      "organization": "kre"
+    }
+{{- end }}
```
Comment (aacecandev) [inline helm/kre/templates/engine/chronograf/config.yml:32]: huehuehue

File: helm/kre/templates/_helpers.tpl [hunk 2]
```diff
@@ -95,3 +95,25 @@ Create MongoDB URI.
...
+{{- end -}}
+
+{{/*
+Create a default fully qualified Kapacitor service name for Chronograph.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "kre-kapacitor.fullname" -}}
+{{- $name := default "kapacitor" .Values.kapacitor.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
```
Comment (aacecandev) [inline helm/kre/templates/_helpers.tpl:119]: newline

**Review by blopezpi — APPROVED** [review]

**Review by aacecandev — APPROVED** [review]

File: helm/kre/templates/engine/chronograf/config.yml (hunk 1 above)
Comment (alpiquero) [reply helm/kre/templates/engine/chronograf/config.yml]: Done

File: helm/kre/templates/_helpers.tpl (hunk 2 above)
Comment (alpiquero) [reply helm/kre/templates/_helpers.tpl]: Done

**Pull request merged**